book/src/drivers.md

```diff
@@ -27,6 +27,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Dell EMC Unity](https://github.com/dell/csi-unity) | `csi-unity.dellemc.com` | v1.0 | A Container Storage Interface (CSI) Driver for [Dell EMC Unity](https://www.dellemc.com/en-us/storage/unity.htm) | Persistent | Read/Write Single Pod | Yes | Snapshot
 [Dell EMC VxFlexOS](https://github.com/dell/csi-vxflexos) | `csi-vxflexos.dellemc.com` | v1.0 | A Container Storage Interface (CSI) Driver for Dell EMC VxFlexOS | Persistent | Read/Write Single Pod | Yes | Snapshot
 [Dell EMC XtremIO](https://github.com/dell/csi-xtremio-deploy) | `csi-xtremio.dellemc.com` | v1.0 | A Container Storage Interface (CSI) Driver for [Dell EMC XtremIO](https://www.dellemc.com/en-us/storage/xtremio-all-flash.htm) | Persistent | Read/Write Single Pod | Yes | Snapshot
+[Diamanti](https://diamanti.com/use-cases/container-network-and-storage-new/) | `dcx.csi.diamanti.com` | v1.1 | A Container Storage Interface (CSI) Driver for Diamanti Spektra | Persistent | Read/Write Single Pod | Yes | Snapshot, Expansion, Raw Block, Cloning
```

Comment:
Can you reference a page that has instructions on how to install/use the csi driver?

Reply:
This is a proprietary driver. Is public-facing documentation/availability a requirement?

Reply:
Yes, the purpose of this listing is to provide technical resources to users that are looking for how to use the driver.

Reply:
I see that some of the other drivers reference documentation that requires authentication. I am happy to link to docs that are on our customer-only support site. Would that be ok @msau42?

Reply:
Yes that is fine. Thanks!

Reply:
Ok - I am going to close this PR for the time being. I will revisit in the next few weeks.